tests/rules/pgsql-7000/test.rules

```diff
@@ -0,0 +1,2 @@
+alert pgsql any any -> any any (msg:"Pgsql table select"; content:"select"; nocase; sid:1;)
+alert pgsql any any -> any any (msg:"Pgsql table delete"; content:"delete from"; nocase; sid:2;)
```

Comment:
Why do we test these 2 rules ? They look similar

Reply:
Wanted to see different transactions being detected.

Reply:
(And then used the same rules from the test in this test)